pkg/config/config.go

```diff
@@ -77,7 +81,11 @@ func Load(path string, netMode netmode.Magic, relativePath ...string) (Config, e
 // be updated based on the provided relative path.
 func LoadFile(configPath string, relativePath ...string) (Config, error) {
 	if _, err := os.Stat(configPath); os.IsNotExist(err) {
-		return Config{}, fmt.Errorf("config '%s' doesn't exist", configPath)
+		if filepath.Dir(configPath) != DefaultConfigPath {
+			return Config{}, fmt.Errorf("config '%s' doesn't exist", configPath)
+		}
+		configBaseName := strings.TrimSuffix(filepath.Base(configPath), filepath.Ext(configPath))
```

Comment:
Remove it, it's an unnecessary action that may be avoided. Make the `getEmbeddedConfig` handle the whole path without any trims, i.e. compare with `fmt.Sprintf("%s/protocol.%s.yml", DefaultConfigPath, netmode.Mainnet)` and etc.